src/App.tsx

```diff
@@ -15,17 +21,38 @@ function App() {
     console.log('genre selected', genre);
   }
 
+  const handleSortChange = (sort: string) => {
+    console.log('sort changed', sort);
+  }
+
+  const handleTileClick = (id: string) => {
+    console.log('movie clicked', id);
+  }
+
+  const handleEditClick = (id: string) => {
+    console.log('Edit movie clicked', id);
+  }
+
+  const handleDeleteClick = (id: string) => {
+    console.log('Delete movie clicked', id);
+  }
+
   return (
-    <div className="App">
+    <div className="App p-5">
       <div className="App-header container mb-2 d-flex flex-column">
         <div className="App-search-container p-5">
           <SearchForm onSearch={handleSearch}/>
         </div>
       </div>
-      <div className="App-body container">
-        <GenreSelect genres={genres} initialSelectedGenre={genres[0]} onSelect={handleGenreSelect}/>
-        <div className="m-3 mt-5">
-          <Counter initialValue={1}/>
+      <div className="App-body container px-5">
+        <div className="d-flex justify-content-between">
+          <GenreSelect genres={genres} initialSelectedGenre={genres[0]} onSelect={handleGenreSelect}/>
+          <SortControl initialSort="title" onSortChange={handleSortChange}/>
+        </div>
+        <div className="d-flex justify-content-between flex-wrap py-3">
+          <MovieTile movieInfo={movies[0]} onClick={handleTileClick} onEdit={handleEditClick} onDelete={handleDeleteClick}></MovieTile>
+          <MovieTile movieInfo={movies[1]} onClick={handleTileClick} onEdit={handleEditClick} onDelete={handleDeleteClick}></MovieTile>
+          <MovieTile movieInfo={movies[2]} onClick={handleTileClick} onEdit={handleEditClick} onDelete={handleDeleteClick}></MovieTile>
```

Comment:
Could you please use `.map` here instead?
Also, for the components without children you can write like so `<MovieTile ... />` instead of `<MovieTile ...></MovieTile>` and this applies in multiple places in this PR